mantle/cmd/kola/testiso.go

```diff
@@ -456,7 +460,11 @@ func testPXE(ctx context.Context, inst platform.Install, outdir string, offline
 	if err != nil {
 		return errors.Wrapf(err, "running PXE")
 	}
-	defer mach.Destroy()
+	defer func() {
+		if err := mach.Destroy(); err != nil {
+			panic(err)
```

Comment:
I'd just log this one I think instead of panicking so that the other `defer`s can still run.